### PR TITLE
[BYOC][ACL] Add maximum support for float32

### PR DESCRIPTION
**Pull request by giuseros**
* Added the code generation flow in arm_compute_lib.py
* Added the runtime calls in acl_runtime.cc

Change-Id: I69c5522f05a46c1dd235da5d57fe499134de0425

### CODE REVIEW

**Comment by giuseros** [comment]
@lhutton1  Do you mind reviewing it?

File: tests/python/contrib/test_arm_compute_lib/test_maximum.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    if skip_runtime_test():
+        return
+
+    device = Device()
+    np.random.seed(0)
+
+    for dtype, low, high, atol, rtol in [
+        ("float32", -127, 128, 0.001, 0.001),
+        ("float32", -1, 1, 0.001, 0.001),
```
Comment (lhutton1) [inline tests/python/contrib/test_arm_compute_lib/test_maximum.py:74]: Curious: why check different data ranges here? is it a check for precision?

File: tests/python/contrib/test_arm_compute_lib/test_maximum.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    ]:
+        inputs = {
+            "a": tvm.nd.array(np.random.uniform(low, high, (100, 100)).astype(dtype)),
```
Comment (lhutton1) [inline tests/python/contrib/test_arm_compute_lib/test_maximum.py:77]: Would be useful to include a test for an asymmetric shape as well i.e. (10, 100). Just to check the ordering of inputs remains the same

File: tests/python/contrib/test_arm_compute_lib/test_maximum.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            outputs.append(build_and_run(func, inputs, 1, None, device, enable_acl=acl)[0])
+
+        verify(outputs, atol=1e-7, rtol=1e-7)
+
+
+def test_codegen_maximum():
+    if skip_codegen_test():
+        return
+
+    shape = (100, 100)
```
Comment (lhutton1) [inline tests/python/contrib/test_arm_compute_lib/test_maximum.py:93]: Same as above

**Review by lhutton1 — COMMENTED** [review]
Thanks! Looks good overall. Could you also add the operator to the table in `docs/deploy/arm_compute_lib.rst` as well :)

**Review by comaniac — COMMENTED** [review]
Overall LGTM. Please address comments from @lhutton1

**Comment by zhiics** [comment]
@comaniac please manage to test your committer permission:)

**Comment by comaniac** [comment]
@giuseros please address the comments from @lhutton1 and we can merge this PR.

**Comment by giuseros** [comment]
Hi @comaniac , @lhutton1 ,
I will have them addressed by tomorrow. 

Thanks,
Giuseppe

File: tests/python/contrib/test_arm_compute_lib/test_maximum.py (hunk 1 above)
Comment (giuseros) [reply tests/python/contrib/test_arm_compute_lib/test_maximum.py:74]: Yes, it is mostly to evaluate float32 numbers in a narrower range

**Review by comaniac — APPROVED** [review]
LGTM. Just wait for the last commit of adding the operator to the doc.

**Review by lhutton1 — APPROVED** [review]
LGTM, modulo the doc fix

**Comment by comaniac** [comment]
Thanks @giuseros @lhutton1 @zhiics

**Pull request merged**